templates/portal/base.html

```diff
@@ -85,6 +85,10 @@
               </div>
             </div>
         </nav>
+        <div class="alert alert-info alert-dismissible fade show" role="alert">
+            <strong>Development Notice:</strong> The codebase is real, but the database is not. Any signups, accounts, or data entered on this site may be deleted without notice. For now, feel free to play around.
+            <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
```

Comment:
Thanks for the PR. I was trying this locally. I'm thinking the banner should always be there, and not dismissable, eg without the close button.
Could you adjust such that the banner cannot be closed?